test/e2e/testing-manifests/ingress/http/ing.yaml

```diff
@@ -3,6 +3,9 @@ kind: Ingress
 metadata:
   name: echomap
 spec:
+  backend:
+    serviceName: echoheadersx
+    servicePort: 80
```

Comment:
Maybe add a comment to tell folks removing this will break mci test?

Reply:
Done